portfolio.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE HTML>
+
+<html>
+
+<head>
+	<link rel="stylesheet" type="text/css" href="https://necolas.github.io/normalize.css/8.0.1/normalize.css">
+	<link rel="stylesheet" type="text/css" href="portfolio.css">
+	<title>Amyas Gilbert's Portfolio</title>
+	<!-- <meta charset="UTF-8"> -->
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+
+<body>
+
+	<nav>
+		<div class="nav-buttons">
+			<a href="#about">About</a>
+			<a href="#portfolio">Portfolio</a>
+			<a href="#contact">Contact</a>
+		</div>
+	</nav>
+	
+	<header>
+		<img class="tardi-image" src="tardi-dude.png">		
+		<div class="header-text">
+			<h1>Greetings!</h1>
+			<p class="intro-text">My name is Amyas and I'm a Full Stack Developer</p>
+			<p class="peruse">Please peruse my portfolio ✨</p>
+		</div>
+	</header>
+					
+	<div class="about" id="about">			
+		<div class="about-text">
+			<h2>About me</h2>
+			<p>So here is where I write some more stuff about me and my coding experience, what I think I'm particularly good at and interested in.</p> 
+			<p>Perhaps also something about my other work experience up until now. I'll figure this out later.</p>
+		</div>		
+		<img class="profile-pic" src="profile-pic.jpeg">	
+	</div>
+		
+	<div class="portfolio" id="portfolio">			
+		
+		<div class="portfolio-intro">
+			<h2>Portfolio</h2>
+			<p>Here are some nice projects what I've worked on...</p>			
+		</div>
+			
+		<div class="projects-container">			
+				
+			<div class="project">				
+				<div class="thumbnail">
+					<h3>Project title</h3>
+					<img class="project-image" src="space-grade.jpg">
```

Comment:
these images are quite small if you think about what they will be showing. i would also say they should be 100% width on mobile, as should be the pic of you in the about section